dbms/programs/client/Client.cpp

```diff
@@ -563,9 +563,16 @@ class Client : public Poco::Util::Application
         if (is_interactive)
         {
             std::cout << "Connected to " << server_name
-                      << " server version " << server_version
-                      << " revision " << server_revision
-                      << "." << std::endl << std::endl;
+                << " server version " << server_version
+                << " revision " << server_revision
+                << "." << std::endl << std::endl;
+
+            if (VERSION_STRING < server_version)
```

Comment:
We compare them as strings. But it will give wrong result if we compare 19.2 vs 19.11.
We need to collect all version components to a tuple of integers and compare them.

Reply:
Done